integreat_cms/static/src/js/events/event-query-pois.ts

```diff
@@ -48,6 +45,68 @@ const setPoi = ({ target }: Event) => {
     console.debug("Rendered POI data");
 };
 
+const showMessage = (response: any) => {
+    console.log("message will be shown");
+
+    const timeoutDuration = 10000;
+
+    if (response.success) {
+        hidePoiFormWidget();
+        const successMessageField = document.getElementById("poi-ajax-success-message");
+        successMessageField.classList.remove("hidden");
+        setTimeout(() => {
+            successMessageField.classList.add("hidden");
+        }, timeoutDuration);
+    } else {
+        const errorMessageField = document.getElementById("poi-ajax-error-message");
+        errorMessageField.classList.remove("hidden");
+        setTimeout(() => {
+            errorMessageField.classList.add("hidden");
+        }, timeoutDuration);
+    }
+};
+
+// TODO: Update to fit the new concept
+const showPoiFormWidget = async ({ target }: Event) => {
+    const option = (target as HTMLElement).closest(".option-new-poi");
+
+    const poiTitle = option.getAttribute("data-poi-title");
+    const response = await fetch(document.getElementById("show-poi-form-button").getAttribute("data-url"));
+    document.getElementById("poi-form-widget").innerHTML = await response.text();
+    document.querySelector("[data-poi-title]").setAttribute("value", option.getAttribute("data-poi-title"));
+    document.getElementById("show-poi-form-button").classList.add("hidden");
+
+    // Add listeners for save and draft-save buttons
+    document.querySelectorAll("[data-btn-save-poi-form]").forEach((el) => {
+        el.addEventListener("click", async (event) => {
+            event.preventDefault();
+            const btn = event.target as HTMLInputElement;
+            const form = btn.form as HTMLFormElement;
+            const formData = new FormData(form);
+            formData.append(btn.name, btn.value);
+            formData.append("title", poiTitle);
```

Comment:
```suggestion

```

So that the poi will be saved with a new name if the name has been changed in the widget form.